IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -110,6 +112,7 @@ - (instancetype)initWithDataSource:(id<MSIDExtendedTokenCacheDataSource>)dataSou
                               targetMatching:cacheQuery.targetMatchingOptions
                             clientIdMatching:cacheQuery.clientIdMatchingOptions])
             {
+                MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, context, @"(MSIDCredentialCache) cached item failed the filtering check - matchesWithRealm:clientId:familyId:target:requestedClaims:targetMatching:clientIdMatching");
```

Comment:
Looks like we are logging the method signature here and not sure if we want to expose it.

Also if there would be any future changes regarding method, we will have to update here as well. Should we have a more generic way of doing so?

Reply:
Good idea. let me rename it to make it more generic :)